config/backend-config/backend-config.go

```diff
@@ -308,3 +308,10 @@ func getNotOKError(respBody []byte, statusCode int) error {
 	}
 	return fmt.Errorf("backend config request failed with %d%s", statusCode, errMsg)
 }
+
+func (d *DestinationT) GetRouterIdentifier() string {
+	if config.GetBool(fmt.Sprintf("Router.%s.isolateDestID", d.DestinationDefinition.Name), false) {
```

Comment:
Changing this configuration option during runtime shouldn't be supported as it could have undesirable effects with respect to the number of routers we would end up having active.
Thus, we should read this configuration option only once at startup.